packages/desktop-client/src/components/modals/TrueLayerExternalMsgModal.tsx

```
@@ -0,0 +1,226 @@
+import React, { useState, useRef } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { Button } from '@actual-app/components/button';
+import { AnimatedLoading } from '@actual-app/components/icons/AnimatedLoading';
+import { Paragraph } from '@actual-app/components/paragraph';
+import { theme } from '@actual-app/components/theme';
+import { View } from '@actual-app/components/view';
+
+import { type TrueLayerAuthSession } from 'loot-core/types/models';
+
+import { Error } from '@desktop-client/components/alerts';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)